.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  build-and-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Install node packages
+        run: npm install
+
+      - name: Run Eslint
+        if: always()
```

Comment:
just curious, could the `if: always()` be omitted?

Reply:
The main reason I put it there because if `npm install` fails it will not run `ESLint` and `Danger` won't be able to read `ESLint` error if it's not run.

Reply:
should we remove it from other places?

Reply:
If we do that and then when somehow the `Run ESLint` fails Github Actions will just skip all the next steps. The `Danger` process won't run either so it won't be able to make comments in the PR with the lint problems.  By providing `if: always()` in every other step makes sure that it runs even if the previous step fails. This way `Danger` can pick up all the problems and comment accordingly.

Reply:
oh, get it. Thanks for explaining